### ISSUE TITLE
Old 3ds models?

### ISSUE THREAD

**Issue by LeedleFace** [leoetlino/project-restoration#226]
I have an original 2ds. I won't be able to use the shortcuts for the New 3ds model. Is there a fix for that?

**Comment by leoetlino** [comment]
No, unfortunately there currently isn't. Not sure what I could do to work around the low number of buttons on the original 2DS/3DS.

However, you can still use the D-Pad shortcuts, just not the second level button mappings that rely on the ZR button.
